recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,20 @@
+cudnn:
+  - 7.6.5
```

Comment:
Added here as a workaround until PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/375 ) lands.

Reply:
So it can be safely removed now?

Reply:
Yeah. We have to wait a little bit for `conda-forge-pinning` to be copied over to the CDN (~20mins) and then update the bot (~2mins). After that we can drop this.

Figure in the meantime we can wait for CI and see if this fixes things or if there is more work to do.

Reply:
Looks like CI is happy with 10.2.

Reply:
Have reverted now 🙂